From: Lennart Poettering <lennart@poettering.net>
Date: Mon, 7 Jul 2014 14:59:06 +0200
Subject: architecture: add string table entries for mips-le archs which were
 missing

(cherry picked from commit 037c26d0aeb750ca9c8d605884ea1db7baecfea8)
---
 src/shared/architecture.c | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/src/shared/architecture.c b/src/shared/architecture.c
index 6cdca4e..dc45f35 100644
--- a/src/shared/architecture.c
+++ b/src/shared/architecture.c
@@ -153,7 +153,9 @@ static const char *const architecture_table[_ARCHITECTURE_MAX] = {
         [ARCHITECTURE_SPARC] = "sparc",
         [ARCHITECTURE_SPARC64] = "sparc64",
         [ARCHITECTURE_MIPS] = "mips",
+        [ARCHITECTURE_MIPS_LE] = "mips-le",
         [ARCHITECTURE_MIPS64] = "mips64",
+        [ARCHITECTURE_MIPS64_LE] = "mips64-le",
         [ARCHITECTURE_ALPHA] = "alpha",
         [ARCHITECTURE_ARM] = "arm",
         [ARCHITECTURE_ARM_BE] = "arm-be",
